index.html

```diff
@@ -59,10 +63,11 @@ <h1>Survey Form</h1>
                 </div>
             </div>
             <div class="row align-content-start">
-                <div class="col col-label">
+                <div class="col-4 col-label font-weight-bold">
```

Comment:
each label is being marked as bold, why not have it universally in css file for each label.